test/e2e/logs/otel/runtime_input_test.go

```diff
@@ -123,8 +123,7 @@ var _ = Describe(suite.ID(), Label(suite.LabelLogsOtel, suite.LabelSignalPull, s
 				g.Expect(bodyContent).To(HaveFlatOtelLogs(ContainElement(SatisfyAll(
 					HaveOtelTimestamp(Not(BeEmpty())),
 					HaveObservedTimestamp(Not(BeEmpty())),
-					HaveLogRecordBody(Not(BeEmpty())),
```

Comment:
Shouldn't we validate the body to contain nothing?

Reply:
checking empty body attribute added